test/xref_runner_SUITE.erl

```diff
@@ -0,0 +1,517 @@
+-module(xref_runner_SUITE).
```

Comment:
I bet you wanted to add something like [this](https://gist.github.com/elbrujohalcon/4761744ffad45d404f5d)